### ISSUE TITLE
VoteGauge

### ISSUE THREAD

**Issue by kimurayu45z** [sunriselayer/sunrise#137]
```
TimeoutError: Transaction with ID 82C41B227B7DCFE5302E4B2843C01F2A197F4F2D8ECC995347A8963EE867F6A3 was submitted but was not yet found on the chain. You might want to check later. There was a wait of 60 seconds.
```

**Comment by Senna46** [comment]
https://testnet.risescan.sunriselayer.io/txs?module=liquidityincentive&perPage=10&currentPage=1

It's not impossible, some users have been able to do it, but currently it usually fails from the frontend.

**Comment by Senna46** [comment]
The problem of transactions remaining unprocessed was resolved in #149 and #150. 
Closed as resolved, although the impact on those already in the network is unknown.
